src/amplitude-client.js

```diff
@@ -1,5 +1,6 @@
 import Constants from './constants';
 import cookieStorage from './cookiestorage';
```

Comment:
are we planning to get rid of the cookieStorage then?

Reply:
Can't in this release.